src/pages/home/_trust_pilot.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import axios from 'axios'
+import TrustLogo from 'images/svg/trustpilot/trust-logo.svg'
+import TrustStar from 'images/svg/trustpilot/trust-star.svg'
+import Stars from 'images/svg/trustpilot/stars'
+import { Flex } from 'components/containers'
+import { Text } from 'components/elements'
+import { Localize } from 'components/localization'
+import device from 'themes/device'
+
+const StyledFlex = styled(Flex)`
+    .trustpilot-widget {
+        flex: none;
+        width: 16rem;
+    }
+`
+
+const StyledLink = styled.a`
+    width: 100%;
+    display: flex;
+    text-decoration: none;
+
+    @media ${device.tabletS} {
+        justify-content: space-between;
+    }
+`
+
+const StarBox = styled(Flex)`
+    flex-direction: column;
+    align-items: center;
+
+    .logo-box {
+        .star {
+            width: 3.9rem;
+            align-self: baseline;
+        }
+        .logo {
+            width: 12rem;
+            margin: 0 0 -1.4rem 0;
+        }
+    }
+
+    @media ${device.tabletS} {
+        justify-content: flex-start;
+        width: fit-content;
+    }
+`
+
+const ScoreBox = styled(Flex)`
+    flex-direction: column;
+    align-items: center;
+
+    .score {
+        margin: 0 0 -0.6rem 0;
+    }
+
+    @media ${device.tabletS} {
+        width: fit-content;
+        flex: none;
+    }
+`
+
+const RatingBox = styled(Flex)`
+    margin: 1.2rem 0 0 0;
+    width: 13.2rem;
+`
+
+const StarRating = styled.img`
+    width: 100%;
+    height: 100%;
+`
+
+export const getRatingData = (rating) => {
+    const rating_str = typeof rating == 'string' ? rating : rating.toString()
+    const rating_data = rating_str.split('.').map((e, k) => (k == 0 ? e : e >= 5 ? 5 : ''))
+    const rate = rating_data[0]
+    const point = rating_data[1] ? rating_data[1] : 0
+    const icon = 'Star' + [rate, point == 5 ? 'Half' : ''].join('')
+
+    return {
+        point,
+        rate,
+        icon,
+    }
+}
+
+const TrustPilotStars = ({ rating }) => {
+    const { icon } = getRatingData(rating)
+
+    return (
+        <RatingBox>
+            <StarRating src={Stars[icon]} />
+        </RatingBox>
+    )
+}
+
+const TrustPilotWidget = () => {
+    const [trust_pilot, setTrustPilot] = useState(null)
+
+    useEffect(() => {
+        const api_key = process.env.GATSBY_TRUSTPILOT_API_KEY
+        const app_name = process.env.GATSBY_TRUSTPILOT_NAME
```

Comment:
Prefixing it with `GATSBY_` will make it available in the browser. You should remove it for this use case.